doc/user_guide/color_usage.rst

```diff
@@ -0,0 +1,1525 @@
+        chart_dict = chart_settings(scheme_name, dict_schemes, continuous)    
+
+        # unpack chart_dict
+        custom = chart_dict['custom']
+        data = chart_dict['data']
+        w = chart_dict['width']
+        h_l = 50
+        h_s = 20
+        cr_l = chart_dict['cornerRadius_large']
+        cr_s = chart_dict['cornerRadius_small']
+        no = chart_dict['no_colors']
+
+        # define color scale and stroke
+        color_scale = chart_color_scale(scheme_name, dict_schemes, continuous, custom, no, method)
+
+        # define how hex-codes can be derived, from datasource or using color scale
+        if single:
+            expr_hex = "datum.hex" if custom and not continuous else "scale('color', datum.i)"
+        elif top:
+            expr_hex = "datum.hex" if custom and not continuous else "scale('concat_0_color', datum.i)"
+        elif bottom:
+            expr_hex = "datum.hex" if custom and not continuous else "scale('concat_1_color', datum.i)"
+
+        # define styling for y-axis
+        my_y_axis = alt.Axis(
+            grid=False,
+            ticks=False,
+            labels=False,
+            domain=False,
+            titleAngle=0,
+            titleAlign="right",
+            titleBaseline="middle",
+        )    
+
+        # determine color pallettes for different color deficiencies
+        normal_vision = chart_normal_vision(data, h_l, w, expr_hex, my_y_axis, cr_l, no)
+        green_blindness = chart_green_blindness(data, h_s, w, expr_hex, my_y_axis, cr_s, no)
+        red_blindness = chart_red_blindness(data, h_s, w, expr_hex, my_y_axis, cr_s, no)
+        monochrome = chart_grayscale(data, h_s, w, expr_hex, my_y_axis, cr_s, no)
+
+        # determine which color pallettes to return
+        if cvd and grayscale:
+            chart_concat = (normal_vision & green_blindness & red_blindness & monochrome & color_scale)
+        elif cvd and not grayscale:
+            chart_concat = (normal_vision & green_blindness & red_blindness & color_scale)
+        else:
+            chart_concat = (normal_vision & color_scale)
+
+        # set configuration on concatenated chart object
+        if single:
+            chart_concat = chart_concat.configure_concat(spacing=1).configure_view(stroke=None)    
+        return chart_concat
+
+
+The basics of color selection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Color selection is an essential aspect of creating clear and informative
+visualizations of scientific data. To make the most effective choices,
+it's important to understand the basics of color perception and theory.
+
+Color perception and theory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We will explain the basics using two colors, ``blue`` and ``orange``: 
+ 
+.. altair-plot::
+    :remove-code:
+
+    percep_schemes = {
+        "two_color": ['blue','orange']
+    }    
+    plot_scheme("two_color", percep_schemes, cvd=False, continuous=False, method='hcl').configure_scale(rectBandPaddingInner=0.1)
```

Comment:
In all the plots in this section, could we remove the "normal vision" label and only include it in the section that talks about different types of color vision? I think the blue and orange rectangles could look better if they were  made a bit smaller as well.

In the later plots, we might also want to consider calling it something like "Standard/reference/perfect color vision" instead of "normal". And likewise change "color blindness" to "color vision deficiencies" which is more descriptive of the partial loss in color vision that most people with this condition have (rather than a complete color blindness which is quite rare)